QuizWhois/Domain/Repositories/Implimintations/BaseRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Domain.Entity.Base;
+using Domain.Repositories.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Repositories.Implimintations
+{
+    public class BaseRepository<TDbModel> : IBaseRepository<TDbModel> where TDbModel : Base
```

Comment:
Since we already use EF another abstraction layer on the top of it seems redundant. Some developers consider repo an anti-pattern so let's double check if we really want to use it

Reply:
https://habr.com/ru/post/335856/